clients/src/main/java/io/strimzi/common/MessageType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.common;
+
+public enum MessageType {
+    text,
```

Comment:
make it all `uppercase` or all `lowercase` maybe?

Reply:
Usually SonarLint suggest to create enum types with `UPPERCASE`.

Reply:
Updated